<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDeploy</class>
 <widget class="QMainWindow" name="AddDeploy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1640</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1640</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1640</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_DestPath">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Files:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination Path:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_SelectSourceFiles">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Select Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_SelectDestinationPath">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Select Path</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_SourceFiles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>591</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
